experimental/Schemes/Auxiliary.jl

```diff
@@ -323,15 +323,15 @@ function _compute_inherited_glueing(gd::InheritGlueingData)
 
   x_img = gens(OO(X))
   x_img = pullback(inverse(iso_X)).(x_img)
-  x_img = OO(UXY).(x_img)
+  x_img = [OO(UXY)(x, check=false) for x in x_img]
   x_img = pullback(gres).(x_img)
   phi = restrict(iso_Y, YX, VYX, check=false)
   x_img = pullback(phi).(x_img)
   gg = SpecMor(YX, XY, hom(OO(XY), OO(YX), x_img, check=false), check=false)
 
   y_img = gens(OO(Y))
   y_img = pullback(inverse(iso_Y)).(y_img)
-  y_img = OO(VYX).(y_img)
+  y_img = [OO(VYX)(y, check=false) for y in y_img]
```

Comment:
```suggestion
  y_img = OO(VYX).(y_img; check=false)
```
You can broadcast with keyword arguments too as far as I remember.

Reply:
I didn't know. Thanks.